front/sssscs/src/register/Register.tsx

```diff
@@ -0,0 +1,90 @@
+import { Box, TextField, Button, Typography } from "@mui/material";
+import { AxiosResponse, AxiosError } from "axios";
+import { RegisterService, User, UserCreateDto } from "./RegisterService";
+import { FieldValues, useForm } from "react-hook-form";
+
+const TryRegister = async (data: FieldValues) => {
+    const dto: UserCreateDto = {
+        'email': data['email'],
+        'name': data['name'],
+        'surname': data['surname'],
+        'password': data['password']
+    }
+    RegisterService.register(dto)
+        .then((res: AxiosResponse<User>) => {
+            console.log(res.data);
+            console.log(res.status);
+        })
+        .catch((err : AxiosError) => {
+            console.error(err.response?.data);
+            console.error(err.response?.status);
+        });
+}
+
+const Register = () => {
```

Comment:
We should agree on whether to add type annotations to _all_ components or _none_ of them. I think the syntax is cubersome:
```ts
const ComponentName: React.FC<{}> = () => {
```
but once our components become complex enough to need state or props, it can help with readability.

@swopl Thoughts?

Reply:
Do we need the types for components? Are they useful anywhere? I prefer the way without them

Reply:
https://github.com/magley/sssscs/pull/2/commits/4fd34a72e41fb190e728d2d2a9cd4284826b6c08